### DISCUSSION TITLE
Support Jalali Date for DatePicker

### DISCUSSION THREAD

**Discussion by MR-Mostafa** [mantinedev/mantine#1827] [Feature requests]
Hello
Thanks for the great project,
In the new version of Mantine, is there any plan to support Jalali Date for DatePicker?

**Comment by rtivital** [comment]
Hi @MR-Mostafa, thanks for kind words, currently we are not planning to add other calendars support to DatePicker component.

**Comment by ShahriarKh** [comment]
These 2 packages are day.js plugins for Jalali calendar. However, I'm not sure if they work correctly because both of them are abandoned.
https://github.com/alibaba-aero/jalaliday
https://github.com/torbatian/dayjs-jalali
Tell me if they worked for you or not. 

(I wish there was native support for Shamsi (Jalali) calendar, but until then, I suggest you to use https://github.com/shahabyazdi/react-multi-date-picker)
